cli/azd/resources/apphost/templates/resources.bicept

```diff
@@ -168,7 +168,7 @@ resource {{bicepName $name}} 'Microsoft.App/containerApps@2023-05-02-preview' =
       containers: [
         {
           image: '{{$value.Image}}'
-          name: '{{$name}}'
+          name: '{{containerAppName $name}}-${resourceToken}'
```

Comment:
We may want to be consistent across the board -- for `AddProject`, we currently do not transform the name either when generated in the container app manifest.